test/factories/communities.rb

```diff
@@ -0,0 +1,24 @@
+FactoryBot.define do
+  factory :community do
+    # Make sure the ranges make sense.
+    lat1 = Random.rand(-90.0..90.0)
+    lat2 = Random.rand(-90.0..90.0)
+    lat1, lat2 = lat2, lat1 if lat2 < lat1
```

Comment:
A more compact version, using sort:

`lat1, lat2 = [Random.rand(-90.0..90.0), Random.rand(-90.0..90.0)].sort`

To avoid duplicating the random number bit, you can use an Array construction with a block, but it's perhaps less readable? Probably a personal preference between these two.

`lat1, lat2 = Array.new(2) { Random.rand(-90.0..90.0) }.sort`

Reply:
Nice, I went with the Array construction with a block.